code/Makefile

```diff
@@ -105,7 +107,7 @@ NOISY=no
 SUMMARIZE_TEX=no
 
 # actual targets
-all: test
+all: test analysis
```

Comment:
I think the analysis shouldn't be generated every build. I don't think the structure of Drasil and development methodology warrants that at the time being. I think analysis should be similar to graphs and generated on CI for the website?

Reply:
The plan was to generate the Data Table (analysis) and post it to the website (probably in a separate step). Regarding whether analysis should be generated every build: what are your thoughts on this, @JacquesCarette and @smiths?

Reply:
I agree with @Mornix.  The analysis doesn't have to be generated with ever build.

Reply:
It should be generated at the same time as the graphs.  See the `graph` target.

Reply:
In that case, should I just remove `analysis` from `all`, and leave `analysis` as a callable target like `graph`?

Reply:
Yes. And make sure that Travis, when doing the nightly deploy, triggers `analysis`, like it already does with `graph`.

Reply:
To do that, do I simply edit `deploy: graphs docs tex doxygen` to `deploy: analysis graphs docs tex doxygen`? Or are there other changes that also need to be made?

Reply:
Yes, I think so.

Reply:
I also made changes to `deploy_lite`, going from this
```makefile
deploy_lite:
	@mkdir -p "$(DEPLOY_FOLDER)"
	@BUILD_FOLDER="$(BUILD_FOLDER)" DEPLOY_FOLDER="$(DEPLOY_FOLDER)" GRAPH_FOLDER="$(GRAPH_FOLDER)" \
	EXAMPLE_DIRS="$(EXAMPLE_DIRS)" MULTI_SRC_DIRS="$(MULTI_SRC_DIRS)" \
	DEPLOY_CODE_PATH_KV_SEP="$(DEPLOY_CODE_PATH_KV_SEP)" MAKE="$(MAKE)" "$(SHELL)" "$(SCRIPT_FOLDER)"deploy_stage.sh
```
to this
```makefile
deploy_lite:
	@mkdir -p "$(DEPLOY_FOLDER)"
	@BUILD_FOLDER="$(BUILD_FOLDER)" DEPLOY_FOLDER="$(DEPLOY_FOLDER)" GRAPH_FOLDER="$(GRAPH_FOLDER)" \
	ANALYSIS_FOLDER="$(ANALYSIS_FOLDER)" EXAMPLE_DIRS="$(EXAMPLE_DIRS)" MULTI_SRC_DIRS="$(MULTI_SRC_DIRS)" \
	DEPLOY_CODE_PATH_KV_SEP="$(DEPLOY_CODE_PATH_KV_SEP)" MAKE="$(MAKE)" "$(SHELL)" "$(SCRIPT_FOLDER)"deploy_stage.sh
```
in adding in the `analysis` folder. Is this change also necessary, or not?

Reply:
Actually, I'm going to pause on the second changes target `deploy_lite` (as they are more relevant to future changes to post the auto-generated analysis to the website.

Reply:
Ok.